components/ui/header.tsx

```diff
@@ -5,41 +5,99 @@ import PearLightLogo from "./PearLight70x70.svg";
+                        <path
+                          fill-rule="evenodd"
+                          clip-rule="evenodd"
+                          d="M8.76512 4.54863L8.7436 4.56796C8.7429 4.56117 8.74206 4.55284 8.74115 4.54304C8.73549 4.54169 8.73049 4.54048 8.72618 4.53941L8.73971 4.52713C8.71027 4.19043 8.62726 2.57318 9.61139 1.47009C10.693 0.257741 13.0997 0.514699 13.1943 0.525314L13.1987 0.525432C13.1987 0.525432 13.1681 3.09028 11.8328 4.05459C10.6403 4.91577 9.08431 4.62144 8.76512 4.54863Z"
+                          fill="black"
+                        />
+                      </g>
+                      <path
+                        d="M12.1637 17.3877C12.1637 19.1766 9.75391 20.4879 9.75391 20.4879C9.75391 20.4879 11.131 19.1766 11.131 17.3877C11.131 15.5988 9.89161 14.889 10.7867 14.5651C11.6818 14.2412 12.1637 15.5988 12.1637 17.3877Z"
+                        fill="#FFFAFA"
+                      />
+                      <defs>
+                        <filter
+                          id="filter0_d_757_1022"
+                          x="7.67095"
+                          y="0.495605"
+                          width="6.58095"
+                          height="6.26796"
+                          filterUnits="userSpaceOnUse"
+                          color-interpolation-filters="sRGB"
+                        >
+                          <feFlood
+                            flood-opacity="0"
+                            result="BackgroundImageFix"
+                          />
+                          <feColorMatrix
+                            in="SourceAlpha"
+                            type="matrix"
+                            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+                            result="hardAlpha"
+                          />
+                          <feOffset dy="1.05366" />
+                          <feGaussianBlur stdDeviation="0.52683" />
+                          <feComposite in2="hardAlpha" operator="out" />
+                          <feColorMatrix
+                            type="matrix"
+                            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"
+                          />
+                          <feBlend
+                            mode="normal"
+                            in2="BackgroundImageFix"
+                            result="effect1_dropShadow_757_1022"
+                          />
+                          <feBlend
+                            mode="normal"
+                            in="SourceGraphic"
+                            in2="effect1_dropShadow_757_1022"
+                            result="shape"
+                          />
+                        </filter>
+                      </defs>
+                    </svg>
+                  </Link>
+                  <Link href={"/about"}>About</Link>
+                </div>
+                <Link href={"https://discord.gg/AKy5FmqCkF"}>Discord</Link>
```

Comment:
instead of hard coding each link we can simply just map it 
```js
 {navLinks.map((link, index) => (
                  <Link key={index} href={link.path}>
                    {link.label}
                  </Link>
                ))}
```

Reply:
ye did it